inc/Engine/Optimization/RUCSS/Frontend/Subscriber.php

```diff
@@ -50,7 +50,7 @@ public function __construct( Options_Data $options, UsedCSS $used_css, APIClient
 	 */
 	public static function get_subscribed_events() : array {
 		return [
-			'rocket_buffer' => [ 'treeshake', 1 ],
+			'rocket_buffer' => [ 'treeshake', 11 ],
```

Comment:
This is more a question than a change request:
Should this be 11 (running just after all the others on the default priority) or 9 (running just before)?
Seems like we had a conversation about needing to run some other optimizations on the CSS after it had been shaken (font-display, etc.)?

Reply:
Just saw @Tabrisrp last reply with optimizations starting at 14. 
What about setting to 12 so we have some space also in between just in case we need to run something in between these 2 operations?

Reply:
I'm thinking we can have warmup at `1`, and treeshake at `12`?

Reply:
Yeah....that should work too. Should I make the change on this PR?

Reply:
I'd say yes, that way it's updated right away